merge_reports.js

```diff
@@ -3,27 +3,42 @@
 
 const { config } = require('./config')
 
+/**
```

Comment:
One line of explanation would be nice here (What does "correct summary" mean?)

Reply:
Yes, both of you with Eric are right. I will fix it.